### DISCUSSION TITLE
Was there a PreMatch screen on Heroes Profile?

### DISCUSSION THREAD

**Discussion by CoffeeKing68** [Heroes-Profile/heroesprofile#1063] [Q&A]
Hello! Awesome project love to use it with Uploader.

I only play occasionally, but I recall the Uploader opening PreMatch screen for ARAM games. 

Looking at the Uploader repo its, trying to hit "https://www.heroesprofile.com/PreMatch/Results?prematchID=<ID?>"
And in the Uploader logs, it reports an ERROR (since that url does not exist).

But this route does not exist on this repo. Was it a hotslogs feature? Would be cool to see it here too!

Thanks

**Comment by Zemill** [comment]
It was a feature of the previous site, before we rewrote the site with this repos code.

With that said, looks like we missed this during the rewrite....I will attempt to add it back in

**Reply by Zemill** [reply]
We are currently working on this

https://github.com/Heroes-Profile/heroesprofile/tree/PreMatch

**Comment by Zemill** [comment]
added
